libraries/standard/http/include/http_client.h

```diff
@@ -27,12 +27,12 @@
 #endif
 
 /**
- * Supported HTTP request methods.
+ * @brief Supported HTTP request methods.
  */
-#define HTTP_METHOD_GET     "GET"              /**< HTTP Method GET. */
-#define HTTP_METHOD_PUT     "PUT"              /**< HTTP Method PUT. */
-#define HTTP_METHOD_POST    "POST"             /**< HTTP Method POST. */
-#define HTTP_METHOD_HEAD    "HEAD"             /**< HTTP Method HEAD. */
+#define HTTP_METHOD_GET     "GET"                  /**< HTTP Method GET. */
```

Comment:
uncrustify is different on everyone's computer lol!

Reply:
I'm using uncrustify 0.67 so I wonder why that is.